ibis/backends/clickhouse/__init__.py

```diff
@@ -140,10 +259,86 @@ def list_tables(self, like=None, database=None):
         databases = list(data[0])
         return self._filter_with_like(databases, like)
 
+    def _apply_limit(self, sql, limit):
```

Comment:
Could we rewrite this to wrap the input ibis expression with an `ops.Limit` instead of working with sqlglot expressions after compilation?

Reply:
Yup, good catch, thanks.